java/src/main/java/vowpalWabbit/learner/VWBase.java

```diff
@@ -36,30 +37,34 @@
      * 3.  Call either {@link System#load(String)} or {@link System#loadLibrary(String)}<br>
      * If a user wishes to use the prepackaged JNI libraries (which is encouraged) then no additional steps need to be taken.
      */
-    protected VWBase(final long nativePointer) {
+    VWBase(final long nativePointer) {
```

Comment:
Increased privacy level to indicate that only classes  in the package can extend this class.